.github/workflows/function_test.yml

```diff
@@ -5,9 +5,9 @@ name: Run tests
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main ]
```

Comment:
Thank you a lot for your help. I chanced "master" to "main" in my .yml manually. Now it works.